packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -87,6 +88,20 @@ export class LayeredRuntimeResolverCreator {
         }
       }
 
+      // If delta marker is encountered, add resolver for `delta` query
+      if (parseMarker('delta', resolverElement.graphqlType.description)) {
```

Comment:
To save having to parse the delta marker every time like this, you could add this configuration value to the ModelDefinition [here](https://github.com/aerogear/graphback/blob/f553235391fd3b6554c0acb751e159cdbe3fe4c8/packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts#L75).

So it could be refactored to `if (resolverElement.deltaSync) {`

Reply:
Please check:
https://github.com/aerogear/graphback/blob/bf46cbf020736b3edfe55b2632a369a736f40270/packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts#L80-L84